analyzers/tests/SonarAnalyzer.UnitTest/Rules/UnusedPrivateMemberTest.Fields.cs

```diff
@@ -28,7 +27,7 @@ public partial class UnusedPrivateMemberTest
     {
         [TestMethod]
         public void UnusedPrivateMember_Field_Accessibility() =>
-            OldVerifier.VerifyCSharpAnalyzer(@"
+            builder.AddSnippet(@"
```

Comment:
To consider, I'm not sure how much work that would be:
I'm not happy with all these snippets, it's hard to maintain. Can they be easily moved to existing TestCases file? Where class name should mark the intention of the test.

Reply:
It would be a big change as the existing test case file is also used in the code fix provider tests. I suggest to open an "improvement" issue for consolidating the test cases. WDYT?

Reply:
This probably doesn't deserve an issue. You can put it in your notes and do it if you have some spare time alter